msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -200,7 +201,11 @@ The following list details the various user types along with their accessibility
     > [!NOTE]
     > Anonymous users inherit the global default user-level app permission policy. For more information, see [manage Apps](/microsoftteams/non-standard-users#anonymous-user-in-meetings-access).
 
-A guest or anonymous user can't add, remove, or uninstall apps.
+> [!NOTE]
+>
+> * If a guest user is a part of one-on-one or group chat, then apps cannot be added by anyone in the chat or meetings.
```

Comment:
@v-npaladugu , please remove this line, just received confirmation about guest user experience. They can interact with tabs but cannot add, update or remove them.